src/pages/Facility/billing/account/components/ChargeItemActions.tsx

```
@@ -0,0 +1,135 @@
+            <MoreHorizontal className="size-4" />
+          </Button>
+        </DropdownMenuTrigger>
+        <DropdownMenuContent align="end">
+          <DropdownMenuItem
+            asChild
+            disabled={item.status !== ChargeItemStatus.billable}
+          >
+            <div
+              className="flex items-center cursor-pointer"
+              onClick={() => onEdit(item)}
+            >
```

Biome (2.3.14)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)